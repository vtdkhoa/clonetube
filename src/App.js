import React, { Component, Fragment } from 'react'
import HeaderNav from './containers/HeaderNav/HeaderNav'
import SideBar from './containers/SideBar/SideBar'

class App extends Component {
  render() {
    return(
      <Fragment>
        <HeaderNav/>
        <SideBar/>
      </Fragment>
    )
  }
}

export default App