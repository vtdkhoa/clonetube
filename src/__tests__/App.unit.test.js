import React from 'react'
import App from '../App'
import { Provider } from 'react-redux'
import { shallow } from 'enzyme'
import configureStore from 'redux-mock-store'

const mockStore = configureStore()
const store = mockStore({})

describe('App', () => {
  test('renders', () => {
    const wrapper = shallow(
      <Provider store={store}>
        <App/>
      </Provider>
    )
    expect(wrapper).toMatchSnapshot()
  })
})