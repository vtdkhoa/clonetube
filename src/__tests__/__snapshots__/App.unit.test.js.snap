// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders 1`] = `
<App
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
/>
`;
